students/guillaume/quarter_2_project/leankitbackend/leankitbackend/leankitcards.py

```diff
@@ -0,0 +1,297 @@
+from collections import defaultdict
+import types
+from datetime import datetime
+
+'''
+Maybe define a Project class
+'''
+
+lanes_current_sprint = ['Current Sprint Backlog', 'Doing Now', 'Under Review',
+                        'Finished As Planned']
+
+lanes_Backlogs_next_sprint = ['Backlog', 'Next Sprint Planning']
+
+
+# Attention sub columns of recently finished / ready to archived has the
+# same name
+lanes_Finished = ['Finished As Planned', 'Started but not Finished',
+                  'Discarded or Obsolete']
+
+
+class Leankit_Cards:
```

Comment:
just a style issue -- usual use either capitalization or underscores, but not both:

`LeankitCards`